conf/mini_provisioner/v2/ha.cleanup.tmpl

```diff
@@ -0,0 +1,22 @@
+cluster:
+  TMPL_CLUSTER_ID:
+    name: TMPL_NAME
+    network:
+      management:
+        virtual_host: TMPL_VIRTUAL_HOST
+cortx:
+  software:
+    corosync:
+      secret: TMPL_SECRET
+      user: TMPL_USER
+server_node:
+  TMPL_MACHINE_ID:
+    cluster_id: TMPL_CLUSTER_ID
+    network:
+      data:
+        private_fqdn: TMPL_PRIVATE_FQDN
+      management:
+        interfaces: TMPL_INTERFACES
+        netmask: TMPL_NETMASK
+    s3_instances: TMPL_S3_INSTANCES
```

Comment:
We can hard code this to 11 or confirm with S3 team and get a fixed value. Changing this for different nodes will be a risk. Provisioner will guarantee.

Reply:
Changed to 11.

Reply:
@indrajitzagade Have you talked to S3 team. This number may differ for vm and hw. We will be creating resources on first node only, so whatever value is there on first node we can pick that.

Reply:
@ajaysrivas Not now. But I had a discussion with S3 team during single VM provisioning work. I think the concern was whether we will be able to create 11 instances on VM due to limited resources on the VM. We did not face any issues with 11 I believe.

Reply:
Changed to 1 after discussion in "Design and Integration" meeting. We may need a lower value in order to run IOs or so, hence changed.